### ISSUE TITLE
[BUG] [Node] [FileSystemPaths] tiddler file paths are only set at wiki boot

### ISSUE THREAD

**Issue by joshuafontany** [Jermolene/TiddlyWiki5#4615]
**Describe the bug**
If you have a `$:/config/FileSystemPaths` tiddler set, such as _below_ while running on Node.js
```
[is[system]removeprefix[$:/]addprefix[_system/]]
[has[draft.of]addprefix[drafts/]]
[!is[system]!has[draft.of]tag[Glossary]addprefix[glossary/]]
[!is[system]!has[draft.of]addprefix[wiki/]]
```
Then any tiddler which exists when the wiki is booted will get a `fileInfo.filePath`, but then if the tiddler gains or loses the `Glossary` tag, it will not move to the folder as defined on that line of `$:/config/FileSystemPaths`.

If the Title of the tiddler is changed, the old tiddler is correctly deleted and a new tiddler is created that matches and then is filed in the correct folder (either moving into or out of the `glossary/` directory).

**Expected behavior**
The *.tid file should be deleted and re-saved into the correct path if it matches a new filter after it has been changed.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10 Node.js
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] TW 5.1.22 & TW 5.1.23-pre

**Additional context**

**Comment by Jermolene** [comment]
Hi @joshuafontany that sounds reasonable, I'd welcome a PR if you have any ideas for the fix?

**Comment by joshuafontany** [comment]
Yeah, I'm working on a solution, but debugging the callback functions via VSCode' Node.js options is a headache atm... (breakpoints are not getting hit). Might have something to hare this weekend.

**Comment by Jermolene** [comment]
Thanks @joshuafontany

**Comment by joshuafontany** [comment]
PR here: https://github.com/Jermolene/TiddlyWiki5/pull/4630

**Comment by joshuafontany** [comment]
@Jermolene I believe this can be closed with the 5.1.23 fix. Also cleaning up the callbacks in my latest PR.

**Comment by pmario** [comment]
@joshuafontany ... You did open the issue, so you are able to close it, if you want.
